changelogs/changelog.yaml

```
@@ -3039,3 +3039,19 @@ releases:
     - sanity-boto3.yml
     - sanity-simple.yml
     release_date: '2024-05-16'
+  8.0.1:
+    changes:
+      bugfixes:
+      - backup_plan_info - Bugfix to enable getting info of all backup plans (https://github.com/ansible-collections/amazon.aws/pull/2083).
+      - ec2_instance - do not ignore IPv6 addresses when a single network interface
+        is specified (https://github.com/ansible-collections/amazon.aws/pull/1979).
+      - s3_object - fixed issue which was causing ``MemoryError`` exceptions when
+        downloading large files (https://github.com/ansible-collections/amazon.aws/issues/2107).
+      release_summary: This release includes some bug fixes for the `s3_object`, `ec2_instance`
```

```suggestion
      release_summary: This release includes some bug fixes for the ``s3_object``, ``ec2_instance``
```

This should be double back-ticks (code) rather than single back-ticks (links).  Probably better handled as a separate PR we can merge into main and stable-8

---

Opened https://github.com/ansible-collections/amazon.aws/pull/2126 that I will rebar after this one is merged.